index.js

```diff
@@ -22,7 +22,7 @@ module.exports = function(options) {
       paths.push(path);
     }
 
-    if (file.isBuffer() && path === 'index.html') {
+    if (file.isBuffer() && path === entryPoint) {
```

Comment:
There is already a default value (`index.html`) so the examples don't need any change.